src/Serilog.Sinks.Elasticsearch/LoggerConfigurationElasticSearchExtensions.cs

```diff
@@ -56,5 +56,35 @@ public static class LoggerConfigurationElasticsearchExtensions
 
             return loggerSinkConfiguration.Sink(sink, options.MinimumLogEventLevel ?? LevelAlias.Minimum);
         }
+
+        /// <summary>
+        /// Overload to allow basic configuration through AppSettings.
+        /// </summary>
+        /// <param name="loggerSinkConfiguration">Options for the sink.</param>
+        /// <param name="uri">A URI for the Elasticsearch instance</param>
+        /// <param name="indexFormat"><see cref="ElasticsearchSinkOptions.IndexFormat"/></param>
+        /// <param name="templateName"><see cref="ElasticsearchSinkOptions.TemplateName"/></param>
+        /// <returns>LoggerConfiguration object</returns>
+        public static LoggerConfiguration Elasticsearch(
+            this LoggerSinkConfiguration loggerSinkConfiguration,
+            string uri,
+            string indexFormat = null,
+            string templateName = null)
+        {
+            var options = new ElasticsearchSinkOptions(new[] { new Uri(uri) });
```

Comment:
Although if we use a string, we might be able to also allow passing in a collection or urls. So do a split on ; or , and pass those in as Uri array.